### ISSUE TITLE
Information (not issue)

### ISSUE THREAD

**Issue by AngeroZero** [Alex-the-666/Ice_and_Fire#3699]
This isn't about a bug its just an information i want,
the information i need is if increasing dragon cavern generation number makes it rarer or more common
im sorry i post this here because its not a issue but the discord is inactive and they won't answer

**Comment by Thermolight** [comment]
TLDR; smaller number should mean more caverns.

The comment above the config line says "one out of this number chance per chunk for generation"
There's more comprehensive ways to describe it, but in short, 1 out of every inputted number of chunks will attempt to generate a dragon cavern.
OR
Smaller numbers increase the chance of them appearing because the 1 is more likely when there's fewer other options.

Also note, just because you input something like 20 doesn't mean there's guaranteed to be one every 20 chunks, it's still random, but over maybe 20,000 chunks generated, you could expect an average of 1 for every 20 chunks. Also, sometimes they fail to generate or can't generate based on the biome.

**Comment by AngeroZero** [comment]
if the world has been already generated will it generate other caves with the new settings? 
i mean in new chunks

**Comment by Thermolight** [comment]
I believe it will, but generating a new world is the only guarantee. Old worlds usually still obey changes to config so I'd be surprised if it didn't work, but the only way to know is to check.
